win32ss/user/ntuser/main.c

```diff
@@ -815,6 +815,12 @@ ExitThreadCallback(PETHREAD Thread)
     if (ptiCurrent->KeyboardLayout)
         UserDereferenceObject(ptiCurrent->KeyboardLayout);
 
+    if (ptiCurrent->spDefaultImc)
+    {
+        UserDereferenceObject(ptiCurrent->spDefaultImc);
+        ptiCurrent->spDefaultImc = NULL;
+    }
```

Comment:
I think this way it will be dereferenced only a single time, am I right? :thinking: